reseq_ros2/scripts/generate_configs.py

```diff
@@ -0,0 +1,114 @@
+import os
+import yaml
+import sys
+import shutil
+
+# Set the path to the config directory relative to the scripts directory
+config_path = os.path.join(os.path.dirname(__file__), '../config')
+temp_config_path = os.path.join(config_path, 'temp')
+
+# Ensure the temp directory exists
+os.makedirs(temp_config_path, exist_ok=True)
+
+# Function to clear the temp directory 
+def clear_temp_directory(temp_path):
+    if os.path.exists(temp_path):
+        shutil.rmtree(temp_path)
+        os.makedirs(temp_path, exist_ok=True)
+
+# Function to merge multiple YAML files into a single dictionary
+def merge_yaml(file_paths):
+    merged_data = {}
+    for file_path in file_paths:
+        with open(file_path, 'r') as file:
+            data = yaml.safe_load(file)
+            merged_data.update(data)
+    return merged_data
```

Comment:
I do not understand what this function is for and when it is used

Reply:
Yes, you are right. It was probably used in the previous version of the script and I have forgotten to get rid of it.